macaroons/README.md

```diff
@@ -114,3 +114,29 @@ removing all three default macaroons (`admin.macaroon`, `invoice.macaroon` and
 `readonly.macaroon`, **NOT** the `macaroons.db`!) from their
 `data/chain/<chain>/<network>/` directory inside the lnd data directory and
 restarting lnd.
+
+
+## Manage macaroons
```

Comment:
Perhaps call this chapter "Root key rotation"?
I think this PR should make it into `v0.12.0`. But I think we should avoid referencing specific versions in READMEs. I mentioned a specific version in the section above because the user had to do an explicit upgrade action to use the new functionality. If a command just isn't available yet, I think the users should realize they need to update their version to get it.

Reply:
got it. removed all version tag and, the whole upgrading notes.